### ISSUE TITLE
Find a potential candidates for the Praise rewards working group

### ISSUE THREAD

**Issue by kristoferlund** [givepraise/planning#83]

**Comment by franculio** [comment]
@kristoferlund what would be considered a VIP? Is this referring to identifying the people who are going the part of the Rewards / Praise team (like quantifiers, leads, etc.)?

**Comment by kristoferlund** [comment]
@franculio I believe this was referring to making personal invitations to people we really would like to see joining the praise team.

**Comment by franculio** [comment]
Perfect! @brichis maybe you have some suggestions of the most active users on governance who could be interested in being part of the praise team?

**Comment by brichis** [comment]
Yes, Justine & Vonnie from the Foundation. Katie and Gonna from the Grants Council.  Michael the host of the community calls. Let me think who else, do you need handles and everything? In which format do you need this? @franculio
